app/src/main/java/com/duckduckgo/app/privacy/ui/PrivacyPracticesViewModel.kt

```diff
@@ -39,42 +41,28 @@ class PrivacyPracticesViewModel : ViewModel() {
         val badTerms: List<String> = emptyList()
     )
 
-    private val viewState = MutableStateFlow(ViewState())
```

Comment:
Removed this and the viewState() function as the initial state is provided with the usage of "stateIn".

Reply:
Nice!